### ISSUE TITLE
7.0.0 broke local auth in sessions.authenticate_jwt

### ISSUE THREAD

**Issue by modosc** [stytchauth/stytch-ruby#109]
we're using `sessions.authenticate_jwt` from `StytchB2B::Client`:

```
      response = b2b_client.sessions.authenticate_jwt @session_jwt,
        max_token_age_seconds: Rails.application.config.stytch_client.max_token_age_seconds,
        session_duration_minutes: Rails.application.config.stytch_client.session_duration_minutes
```

with `6.6.0` this code worked correctly - if `@session_jwt` was expired it would get refreshed, otherwise it would be authenticated locally.

with `7.0.0` this behavior stopped working and _every_ call results in an API call to `/v1/b2b/sessions/authenticate`. i've verified that we're using the newly returned `session_jwt` with subsequent requests.

**Comment by logan-stytch** [comment]
Hey @modosc, let me take a look ASAP. Thanks for the report!

**Comment by logan-stytch** [comment]
This should be fixed in v7.0.2
